isis/src/control/objs/BundleResults/BundleResults.cpp

```diff
@@ -95,6 +95,7 @@ namespace Isis {
         m_rmsYResiduals(src.m_rmsYResiduals),
         m_rmsXYResiduals(src.m_rmsXYResiduals),
         m_rejectionLimit(src.m_rejectionLimit),
+	m_grossRejectionLimit(src.m_grossRejectionLimit),
```

Comment:
indentation got messed up here, should be in 2 more spaces.

Reply:
The spacing looks totally fine in my vim? Not sure how to fix this

Reply:
I am going to retry the spacing, maybe I used a tab instead of spaces?

Reply:
You likely used a tab instead of spaces